### PR TITLE
Replace musicbrainz-vm with musicbrainz-docker

### PR DESCRIPTION
**Pull request by yvanzo**
It follows https://blog.metabrainz.org/2020/05/16/musicbrainz-docker-composes-with-solr-7/

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 Installing MusicBrainz Server
 =============================
 
-The easiest method of installing a local MusicBrainz Server may be to download the
-[pre-configured virtual machine](https://musicbrainz.org/doc/MusicBrainz_Server/Setup),
-if there is a current image available. In case you only need a replicated
-database, you should consider using [mbdata](https://github.com/lalinsky/mbdata).
+The easiest method of installing a local MusicBrainz Server may be to use the
+[MusicBrainz Docker](https://github.com/metabrainz/musicbrainz-docker) Compose
+project, which can be used for web site/service mirror, test, or development.
```
Comment (mwiencek) [inline INSTALL.md:6]:
```suggestion
project, which can be used as a website/service mirror (including for development).
```
maybe? not sure what is meant by test that isn't encompassed by development

Comment (reosarevok) [reply INSTALL.md:6]: "as a website/web service mirror" I guess then?

Comment (mwiencek) [reply INSTALL.md:6]: I thought "website/service" would be understandable enough, but that might be clearer. Mainly just can't bear to see "web site" spelled separately. :P

Comment (yvanzo) [reply INSTALL.md:6]: > maybe? not sure what is meant by test that isn't encompassed by development

Meant to distinguish testing your own app with MB WS/2 (or MB DB), from MBS development.

(The same as in the blog post and the wiki doc.)

Comment (mwiencek) [reply INSTALL.md:6]: Sounds like development too, but ok. :) There are still a couple grammatical issues, but this works too:
```suggestion
project, which can be used for a website/web service mirror, testing, or development.
```

Comment (yvanzo) [reply INSTALL.md]: Applied changes in https://github.com/metabrainz/musicbrainz-server/pull/1526/commits/70b89af38d2456888c5fb84d57b860fedc451514, thanks.

**Pull request merged**